terraform/locals.tf

```diff
@@ -1,3 +1,19 @@
+# The list of available AWS availability zones
+data "aws_availability_zones" "all" {}
+
+# The AWS account ID being used
+data "aws_caller_identity" "current" {}
+
+# ------------------------------------------------------------------------------
+# Retrieve the default tags for the default provider.  These are
+# used to create volume tags for EC2 instances, since volume_tags does
+# not yet inherit the default tags from the provider.  See
+# hashicorp/terraform-provider-aws#19188 for more details.
+# ------------------------------------------------------------------------------
+data "aws_default_tags" "default" {
+  provider = aws
+}
```

Comment:
Why do we do this instead of referencing `var.tags` explicitly?

Reply:
That's what I saw in [other projects](https://github.com/search?q=org%3Acisagov+aws_default_tags&type=code) and I pulled it from the skeleton: https://github.com/cisagov/skeleton-tf-module/blob/f1c81f4e0ec92f988c76def81d26786f5688d086/main.tf#L55-L60

Reply:
This could be a misinterpretation on my part because those projects often have multiple providers, but I figured I would follow precedent.

Reply:
In [a2de9a8](https://github.com/cisagov/cyhy_amis/pull/355/commits/a2de9a86237e768dbb7ab25d9de764e8281d850b) I removed the `provider` to keep in line with the examples I linked.